POEApi.Model/Gear.cs

```diff
@@ -67,5 +70,65 @@ public int NumberOfSockets()
                 return components;
             }
         }
+
+        public override string PobData
```

Comment:
Move this into its own class (even if it's just a static method) and add tests to make sure we convert various `Gear` to strings correctly.

Reply:
It's not a method.

Reply:
You can change it into one.  I meant if the resulting class is just a static method.  Doing so is intended to make each class responsible for fewer things (a thing here being "convert a `Gear` to a PoB-compatible string") and make it easier to test.

Reply:
The way it's structure currently is that all items have the ability to provide a POB compatible string.

See:
https://github.com/Stickymaddness/Procurement/blob/ee8c9b0d483a05b012d26038dde74b944bf18842/POEApi.Model/Item.cs#L189

I can't envisage anything which isn't "Gear" being compatible for import into POB but I structured it this way so that other objects can provide their own data.